fastlane/Fastfile

```diff
@@ -648,9 +648,9 @@ desc 'Run source code formatting/linting'
 lane :run_swift_format do |options|
```

Comment:
I think we could now rename this lane. But up to you.

Reply:
As renaming is going to be a no-production-code change, let me merge this one and we can do it in a subsequent PR. what name do you suggest?